src/chrome/content/rules/Samsung.com.xml

```diff
@@ -0,0 +1,157 @@
+-->
+
+<ruleset name="Samsung.com (partial)">
+
+	<target host="opensource.samsung.com" />
+		<!--exclusion pattern="^http://opensource\.samsung\.com/+($|\?|community/(ajax/|community/|community\.do|guest/|proj/)|iframeMain\.do|reception/receptionSub\.do|reception\.do)" /-->
+		<!--exclusion pattern="^http://opensource\.samsung\.com/(?!community/(?:css/|images/|img2?/|img_model/|js2?/|org\.ditchnet\.taglib/|repository/|scmShowFileRevision(?:$|\?)|stylesheet/|usageStatistics\.do|wikiTreeMessage(?:$|\?))|css/|img/|js/)" /-->
+
+	<exclusion pattern="^http://opensource\.samsung\.com/(ajax|community\.do|images|org\.ditchnet\.taglib|popup|scmShowFileRevision|stylesheet|usageStatistics|wikiTreeMessage)?$" />
+	<exclusion pattern="^http://opensource\.samsung\.com/reception(\.do)?/(?!receptionSub)?$" />
+		<!--	To https	-->
+		<test url="http://opensource.samsung.com/community/" />
+		<test url="http://opensource.samsung.com/css/" />
+		<test url="http://opensource.samsung.com/css/genUsersDefault.css" />
+		<test url="http://opensource.samsung.com/img/favicon_ico02.ico" />
+		<test url="http://opensource.samsung.com/img/" />
+		<test url="http://opensource.samsung.com/img2/" />
+		<test url="http://opensource.samsung.com/img_model" />
+		<test url="http://opensource.samsung.com/js" />
+		<test url="http://opensource.samsung.com/js2" />
+		<test url="http://opensource.samsung.com/reception/receptionSub" />
+		<test url="http://opensource.samsung.com/reception/receptionSub.do" />
+		<!--	To http	-->
+		<test url="http://opensource.samsung.com/" />
+		<test url="http://opensource.samsung.com/images" />
+		<!--test url="http://opensource.samsung.com/community.do/" /-->
+		<test url="http://opensource.samsung.com/org.ditchnet.taglib" />
+		<test url="http://opensource.samsung.com/popup" />
+		<test url="http://opensource.samsung.com/reception/" />
+		<test url="http://opensource.samsung.com/reception.do/" />
+		<test url="http://opensource.samsung.com/scmShowFileRevision" />
+		<test url="http://opensource.samsung.com/stylesheet" />
+		<test url="http://opensource.samsung.com/usageStatistics" />
+		<test url="http://opensource.samsung.com/wikiTreeMessage" />
+
+	<rule from="^http://opensource\.samsung\.com/" to="https://opensource.samsung.com/" />
+		
+		<!-- Support page FAQ and other features are broken. Also, their i18n is a lie.
+		     Github issue: https://github.com/EFForg/https-everywhere/issues/309 -->
+		<exclusion pattern="^http://www\.samsung\.com/us/(?:support/|function/)" />
+			<test url="http://www.samsung.com/us/function/" />
+			<test url="http://www.samsung.com/us/support/" />
+
+	<!--	Not secured by server:
+					-->
+	<!--securecookie host="^\.samsung\.com$" name="^(_common_country|_common_lang|COUNTRY_CODE|EXPAND_SCHEDULED|REMEMBERED_DEVICE_CODE_CAN)$" /-->
+	<!--securecookie host="^apps\.samsung\.com$" name="^WMONID$" /-->
+
+	<securecookie host="^(?!shop\.us\.).*\.samsung\.com$" name=".+" />
+
+	<target host="*.samsung.com" />
```

Comment:
It's a bit irritating to have a wildcard target and that much exclusions. I would prefer listing the targets, the you can also get rid of the 'Fully covered domains' list on the top.